packages/table/src/table.tsx

```diff
@@ -543,6 +543,45 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         this.quadrantStackInstance.scrollToPosition(correctedScrollLeft, correctedScrollTop);
     }
 
+    public resizeRowsByApproximateHeight(getCellText: (rowIndex: number, columnIndex: number) => string) {
+        const APPROX_CHAR_WIDTH = 8;
+        const APPROX_LINE_HEIGHT = 18;
+        const NUM_EXTRA_LINES = 1; // extra lines
```

Comment:
if i were using this API i would want access to the above variables as optional parameters

Reply:
Reasonable. 👍